app/src/components/Home.vue

```diff
@@ -0,0 +1,76 @@
+<style lang="scss">
+
+/**
+ *--------------------------------------------------------------------------
+ *	Vue-component for home
+ *--------------------------------------------------------------------------
+ *
+ * 	This is the Vue-component for home.
+ *
+ * 	This file includes the HTML template, sass-stylesheet 
+ * 	and javascript for home.
+ *
+ */
```

Comment:
Bitte solche Kommentare in Zukunft weg lassen / löschen. Die ziehen bzw. zogen sich durch einige Dateien.